runbot_gitlab/runbot_repo.py

```diff
@@ -146,7 +147,16 @@ def set_gitlab_ci_conf(token, gitlab_url, runbot_domain, repo_id):
 
 class RunbotRepo(models.Model):
     _inherit = "runbot.repo"
-    uses_gitlab = fields.Boolean('Use Gitlab')
+
+    uses_gitlab = fields.Boolean(string='Use Gitlab')
```

Comment:
I lately discovered https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field (third bullet point) - this applies to this and the next field